packages/server/src/fhircast/routes.test.ts

```diff
@@ -33,146 +53,197 @@ describe('FHIRCast routes', () => {
   });
 
   test('New subscription success', async () => {
-    const res = await request(app)
-      .post('/fhircast/STU3/')
-      .set('Content-Type', ContentType.JSON)
-      .set('Authorization', 'Bearer ' + accessToken)
-      .send({
+    for (const route of [STU2_BASE_ROUTE, STU3_BASE_ROUTE]) {
```

Comment:
You can use `test.each` if you like.  It's mostly cosmetic, but jest will then report the test outcomes separately rather than a single pass/fail.